__tests__/app.test.js

```diff
@@ -155,7 +151,106 @@ describe('POST /api/articles/:article_id/comments', () => {
+            expect(postedComment).toBe('lorem ipsum');
+        })
+    });
+    it('returns with a status: 400 when all necessary properties are not sent to the server', () => {
+        const comment = {
+            username: 'rogersop',
+        }
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(comment)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.message).toBe("Bad request");
+        });
+    });
+    it('returns with 404 if article does not exist', () => {
+        const comment = {
+            username: 'rogersop',
+            body: 'lorem ipsum'
+
+        }
+        return request(app)
+        .post("/api/articles/87/comments")
+        .send(comment)
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.message).toBe("Not found");
+        });
+    });
+});
+
+
+describe('GET /api/articles/:article_id/comments', () => {
+    it('returns status 200 and an array of corresponding comment objects for the given article_id ', () => {
+        return request(app)
+        .get("/api/articles/3/comments")
+        .expect(200)
+        .then(({body})=> {
+            const comments = body.comments;
+            expect(Array.isArray(comments)).toBe(true);
+            expect(comments).toHaveLength(2);
+            expect(comments).toBeSortedBy("created_at", {
+                descending: true,
+            });
+            if(comments.length > 0){
+                comments.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id");
+                    expect(comment).toHaveProperty("votes");    
+                    expect(comment).toHaveProperty("created_at");    
+                    expect(comment).toHaveProperty("author");    
+                    expect(comment).toHaveProperty("body");    
+                    expect(comment).toHaveProperty("article_id");    
+    
+                });    
+            }        
+        })
+    });
+    it('Returns 404 status and message when article does not exist ', () => {
+        return request(app)
+        .get("/api/articles/77/comments")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.message).toBe('Opps, article does not exist');
+        })
+    });
+    it('Returns 200 status and a message for an article_id that does not correspond to any comments ', () => {
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.message).toBe('No comments found');
+        })
+    });
+    it('Returns 400 status and and a bad request message when passed wrong data type for article_id', () => {
+        return request(app)
+        .get("/api/articles/baddata/comments")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe('Bad request');
+        })
+    });
+
+});
+
+
+describe('POST /api/articles/:article_id/comments', () => {
```

Comment:
I'm not sure what all this code is as it doesn't relate to the /users endpoint that you've submitted the PR for. If you would like feedback for your post etc, please place it in another PR.